apps/system/js/sync_manager.js

```diff
@@ -218,7 +219,11 @@
         case 'disable':
         case 'sync':
           try {
-            Service.request('SyncStateMachine:' + request.name).catch(e => {
+            Service.request('SyncStateMachine:' + request.name).then(() => {
+              return LazyLoader.load('/shared/js/sync/telemetry.js');
```

Comment:
Would maybe be cleaner to do this all from /shared/js/sync/bootstrap.js, so it runs in the sync app and not in the system app, but I guess it doesn't matter so much.